src/stories/index.ts

```diff
@@ -1 +1,2 @@
-import 'storybook-addon-deep-controls'
+// ? this is not required?
+import "storybook-addon-deep-controls";
```

Comment:
This is, indeed, no longer needed, @eliasm307 - the addon used to have a global augmentation for `Parameters` in [0.9.4](https://www.npmjs.com/package/storybook-addon-deep-controls/v/0.9.4?activeTab=code), and I've set up the repo _before_ bumping to 0.9.5 😅